f/flash-b.md

```diff
@@ -0,0 +1,10 @@
+# Flash-b
+
+
+![dan](flash.jpg)
```

Comment:
Dan is not a good alternative text for this Image.
Please change the image filename so that it matches the artist filename.